harmonic/ros2_launch_gazebo.md

```diff
@@ -49,3 +50,53 @@ within this tag. Here's an example for launching Gazebo server:
 In this case the `<gz_server>` parameters are read from the command line. That's
 an option but not strictly necessary as you could decide to hardcode some of the
 values.
+
+### Python
+Python launch files provide more low-level customization and logic compared to XML launch files. 
```

Comment:
Thanks for these changes! The only comment I have is that we now have launch Actions for Gazebo, so the python launch file could be further simplified. cc @caguero

Reply:
Cool! Is this new to Jazzy/Harmonic? Where could I find out how to implement this? Or if it's easier you could send me how you would modify it.

Reply:
I think I could use the style in place here: https://github.com/gazebosim/ros_gz/blob/ros2/ros_gz_sim/launch/gz_spawn_model.launch.py if that's appropriate

Reply:
> Cool! Is this new to Jazzy/Harmonic? 

Yes, well, currently only Rolling, but we will be backporting it to Jazzy.

> Where could I find out how to implement this? 

We currently only have documentation on how to use this from XML (I.e, the `<gz_server>` tag), but the Actions are in
https://github.com/gazebosim/ros_gz/blob/ros2/ros_gz_sim/ros_gz_sim/actions/gzserver.py and https://github.com/gazebosim/ros_gz/blob/ros2/ros_gz_bridge/ros_gz_bridge/actions/ros_gz_bridge.py

Reply:
That's super cool, though since it's still in Rolling for now could I add this example and then use the gz_server action once it gets merged into Jazzy? Still not entirely sure the format of it in the Python launch system.

Reply:
That works for me.